File: codeWar/javaScript/Replace With Alphabet Position.js
//https://www.codewars.com/kata/546f922b54af40e1e90001da/solutions/javascript
function alphabetPosition(text) {
  const alpha = "abcdefghijklmnopqrstuvwxyz";
  let t = text.split("");
  let result = "";
  t.forEach((v)=>{
    let a = v.toLowerCase();
    if (!/[a-z]/.test(a)){
      return 
    }
    let p = alpha.match(a);
    result += p.index + 1+" ";
  })
  return result.trim();
}

File: codeWar/rust/Replace With Alphabet Position.rs
//https://www.codewars.com/kata/546f922b54af40e1e90001da/train/rust
fn alphabet_position(text: &str) -> String {
    // Code here...
    let mut result = "".to_owned();
    for i in text.chars(){
        let j = i.to_digit(36);
        if j == None{
            continue
        }
        let t = j.unwrap();
        if t >= 10 && t<=36{
            result = format!("{} {}",result,t-9);
        }
    }
    result.trim().to_string()
}
